### PR TITLE
chore: refactors balance end-points

### PR DESCRIPTION
**Pull request by jrabbott**
### Context
Refactors balance end-points to align with Census and Income

### CODE REVIEW

File: platform/.run/docker_ Compose Deployment.run.xml
Comment (WolfyUK) [inline platform/.run/docker_ Compose Deployment.run.xml:1]: This is used by Rider multi-project run config. Was it causing you issues?

File: web/tests/Web.Tests/Infrastructure/GivenAnInsightApi.cs [hunk 1]
```diff
@@ -22,17 +22,6 @@ public async Task GetSchoolFinancesShouldCallCorrectUrl()
         VerifyCall(HttpMethod.Get, "api/school/123213");
     }
 
-    [Fact]
```
Comment (WolfyUK) [inline web/tests/Web.Tests/Infrastructure/GivenAnInsightApi.cs:25]: Are `BalanceApi` tests going to be included at a later date?

Comment (jrabbott) [reply web/tests/Web.Tests/Infrastructure/GivenAnInsightApi.cs:25]: Yep - going to do a sweep on tests over the coming weeks,

File: platform/.run/docker_ Compose Deployment.run.xml
Comment (jrabbott) [reply platform/.run/docker_ Compose Deployment.run.xml:1]: Re-add file

**Review by WolfyUK — APPROVED** [review]

**Pull request merged**